pkg/cmd/secrets.go

```diff
@@ -634,6 +643,7 @@ func init() {
 	secretsDownloadCmd.Flags().String("name-transformer", "", fmt.Sprintf("(BETA) output name transformer. one of %v", validNameTransformersList))
 	secretsDownloadCmd.Flags().String("passphrase", "", "passphrase to use for encrypting the secrets file. the default passphrase is computed using your current configuration.")
 	secretsDownloadCmd.Flags().Bool("no-file", false, "print the response to stdout")
+	secretsDownloadCmd.Flags().Duration("dynamics-expire", 0, "dynamic secrets will expire after specified duration, (e.g. '3h', '15m')")
```

Comment:
Does it make sense for this to be a runtime option? As opposed to using whatever the dynamic has been configured for. (if we do keep it, I think `--expire-dynamics` sounds better)

Reply:
I think long-term, we'll allow users to set a max and default during setup of the dynamic secrets. I think there is some value in letting you set it on the CLI for context-specific reasons. For example, I could imagine an AWS user that one CI job will only need for 5 min max (restart service) vs 1 hour max (large bundle upload).

Reply:
Wondering if this would be more appropriate as a follow up item, once the use case has established itself a bit more. Maybe it's already clear that we need this though. Will defer to you.

Reply:
Good looking out -- I do think we've got some clear use cases for this. My gut says ship it